nixos/modules/services/misc/paperless.nix

```diff
@@ -160,32 +163,29 @@ in
       description = lib.mdDoc "Web interface port.";
     };
 
-    # FIXME this should become an RFC42-style settings attr
-    extraConfig = mkOption {
-      type = types.attrs;
+    settings = mkOption {
+      type = lib.types.submodule {
+        freeformType = with lib.types; attrsOf (let
+          typeList = [ attrs bool int str path package ];
+        in oneOf typeList + (listOf typeList));
```

Comment:
```suggestion
        in oneOf typeList ++ (listOf typeList));
```

Reply:
I was working locally on getting this fixed but got distracted. :)